CHANGELOG.md

```diff
@@ -110,6 +110,7 @@ The following plugins were added:
 - Encounter: GetResetTime()
 - Encounter: SetResetTime()
 - Events: GetCurrentEvent()
+- Events: ON_ITEM_EVENT_{BEFORE|AFTER}
```

Comment:
Minor, this should be added to `Events: Added ItemInventory{Open/Close}, ItemInventory{Add/Remove}Item and ScrollLearn events to ItemEvents` under the `Added` header in the changelog.